notebooks/Presentation.ipynb

```diff
@@ -0,0 +1,470 @@
yAG5NBNKjADNbxd+3JjJpJCIqIDZJpRIxZMiQQs8rk8mwY8eOYoyGiIj+cx4uA4zMABNrxf9NrIHsfz93cDNoaEREbxImjFQi/Pz8Cj0vR0klIqIiS3mqqFlUZmwFpIQaJBwiojcVE0YqETt37jR0CERE9F9mXUfRDFWsYQSAnFTAurbBQiIiehMxYaQS0aZNG0OHQERE/2XyGYo+i9lQ1CzmpAK5mYrPiYhIbxz0hoiIiN4+Dm5As7WAhQOQGa/4u9na4uu/GOEB/OUKHKmj+DvCgIPpPPcATrkCe+so/n5ewFhCPYDDrsC2Oqh1fZji/0RE/2LCSERERG8nBzfgPS/goyeKv4szWfSbCKT9OwJrWpTi/4ZIGp97AFcnAin/xpISpfi/vkljqAdw8d/5zW1hnPlK8X8mjUT0LzZJpRKxdu1aAMCgQYNQsWJF6f/6kMlkmDBhQkmFRkREVDT3tYzAen8Z4FjKI7De+TcW039jMbUGsv79vJYesdxcBhj///yCsSVgnKP4vDZHkyUiJoxUQtauXQuZTIaPP/5YShhlMhkEQdA6jzidCSMREZVpyVpGYE0OLf1YkjTEYmIFJOkZy+ungLmG+V/rOT8RvfWYMFKJmDBhAmQyGSpVqqTyfyIiojeeTR1FM9S8I7Da1C79WMrVUTQnNVWKJTsVKKdnLOW1zF9ez/mJ6K3HhJFKxKRJk3T+n4iI6I3VaIaiz2LeEVgbGWAE1iYzFH0Ws6CoGcz+N5YmesbScoaizyIU88ty0v7/cyIicNAbItLTLY9Q/Ox6BFPqbMfPrkdwyyPU0CERERmGoxvQZi1g+e8IrJYOiv+Xdv9FQNFPscNawPrfWKwdFP/Xp/8ioOin2OXf+TPikWNWRfF/9l8kon+xhpEMIjc3FwkJCQCAihUrwsiIzy7Kslseodgx8W+YmBnD2tYCCVGp2DHxb2At0MyttqHDIyIqfY5uhkkQNanlpn+CqEltNylBfB4YCOfazsUUGBG9DZgwUqm6fPkyduzYgZs3byIjIwMAYG5ujpYtW2LIkCF47733DBwhaXJ6mT9MzIxhbm0KANLfp5f5M2EkIiIieosxYaRS8/PPP8Pd3V0aKVWsVUxPT8fVq1dx7do1DBw4EHPmzDFkmKRBzNPXsLa1UPnMzMoEMaFJBoqIiIiIiEoDE0YqFUeOHMGuXbtgbW2NYcOGoXfv3nBwcAAAREVF4cSJE9i+fTt2794NZ2dnfPbZZwaOmJTZ1SmPhKhUqWYRADJTs2FXu5wBoyIiIiKiksaOY1Qq3N3dYWJigm3btmHSpEmoVasWTE1NYWpqilq1amHixInYunUrjI2NsWfPHkOHS3l8PKMFsjNzkJGSBUEQkJGShezMHHw8o4WhQyMiIiKiEsSEkUrF48eP0bp1azRp0kRrmSZNmqB169Z4/PhxKUZG+mjmVhtD176Hig5WSInPQEUHKwxd+x77LxIRERG95dgklUqFpaUlbG1t8y1na2sLc3PzUoiICqqZW20miERUdrz0AJ4sA1KfAlZ1gLozAPsyMmopEdFbhDWMVCpatGiBe/fuSQPeaCIIAu7du4cWLdjMkYiIdHjpAdyfCKRHAaa2ir/vT1R8/l/k7wHMcwXG11H87f8fXQ9EVCKYMFKpmDx5Ml68eIElS5YgKytLbXp2djaWLl2KFy9eYPLkyQaIkIiI3hhPlgEyM8DEGpDJ/v3bTPH5f42/B7BlIpAQBdjYKv7eMpFJIxEVGzZJpRJx7Ngxtc/69OmDnTt3wtPTE927d0eNGjUAABEREfD09MSLFy/w1VdfITg4GM7OfGkwERFpkfpUUbOozNgKSA01SDgGdWIZYGoGmFsr/i/+fWIZ0IJNdImo6JgwUomYNWsWZDKZ2ueCIODFixfYuXOn2ucAsG/fPuzbtw+ffvppaYRJRERvIqs6imaoJtb//1lOKmBV22AhGczLp4qaRWVmVkBMqEHCIaK3DxNGKhGffvqpxoTxTfPw4UP07dsXWVlZqF+/Pk6dOqW17JMnT7Bu3Tr4+PggISEBdnZ26Ny5MyZMmAB7e/tSjJqI6C1Xd4aiz2I2FDWLOamAkKn4vKTd9gDOLANingJ2dYCPZgBNDViTZ19H0QzVXCl5zkwF7GobLCQierswYaQSsWTJEkOHUGTZ2dmYNWsWsrOz8y3r5+eH0aNHIz09HY0aNULr1q0RFBSEffv24a+//sKePXtQp06dUoiaiOg/wN4NwNp/R0kNVdQslsYoqbc9gJ0TARMzwPrf/oI7JwJD1houafxkhqLPIqCoWcxMBbIyFZ8TERUDDnpDpMWGDRtw//59DBgwQGe51NRUTJ06Fenp6Zg7dy6OHDmCVatWwcPDAyNGjEBcXBymTZumc4RYIiIqIHs3oK0X4PpE8XdpvFLjzDJFsihkA7EhQPwTIPkFcGhWyX+3Ni3cgJFrgYoOQEq84u+Ra9l/kYiKDRNGMghBEBAXF4e4uDjk5uYaOhw1QUFB2LBhAz788EN0795dZ9kjR44gJiYGLi4uGDRokMq06dOno1atWrh//z4uX75ckiETEZHogQew1hVYUEfx94NiGjE05imQmwUkPAdysgAjE0DIASIfAMEGHJW0hRswzwv444nibyaLRFSMmDBSqbp69SpGjhyJ5s2bo0OHDujQoQNatGiBkSNH4urVq4YODwCQlZWFWbNmwdraGj/99FO+5c+fPw8A6NWrl9o0Y2NjfPTRRyrliIioBD3wAA5PBJKiACtbxd+HJxZP0mhXB3gdBciMFH8AQABgbgZc/g++0oOI/hOYMFKpWb16NUaNGoWrV68iPT0dMpkMMpkM6enpuHr1KkaNGoU1a9YYOkysX78egYGBmD17NqpUqZJv+cDAQABA48aNNU4XP3/w4EHxBUlERJp5LQMyM4HoCODpPcXfmZmKz4vqoxlAdiaQKwCCAOTkKP52cADiQ4u+fCKiMoiD3lCpuHz5MtatWwdLS0sMGjQ
```

Comment:
> __[Secret Scan]__ It may be dangerous to commit the AWS secret access key. ([view](https://sider.review/gh/repos/262151890/pulls/77#issue-6043044))
>
> <details>
>
> AWS secret access key is an important credential that allows you to use AWS programmatically.
> Once this leaks, an attacker can access and operate your AWS resources.
>
> See also:
> * https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html
>
> | Rule | Severity |
> | ---- | -------- |
> | `sider.secrets.aws.secret_access_key` | `warning` |
>
> You can [close this issue](https://sider.review/gh/repos/262151890/pulls/77#issue-6043044) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When this is not an AWS secret access key.